Casks/hex-fiend.rb

```diff
@@ -4,11 +4,14 @@
 
   url "https://github.com/ridiculousfish/HexFiend/releases/download/v#{version}/Hex_Fiend_#{version.major_minor}.dmg",
       verified: "github.com/ridiculousfish/HexFiend/"
-  appcast "https://github.com/ridiculousfish/HexFiend/releases.atom"
   name "Hex Fiend"
   desc "Hex editor focussing on speed"
   homepage "https://ridiculousfish.com/hexfiend/"
 
+  livecheck do
+    strategy :github_latest
```

Comment:
```suggestion
    url :cask_url
    strategy :github_latest
```

`url` is required in a `livecheck` block.